### PR TITLE
footer final version

### PR DESCRIPTION
**Pull request by Igor-Pysak**
[DEMO LINK](https://Igor-Pysak.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://Igor-Pysak.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,52 @@
...
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
```
Comment (nazarmatsevych) [inline src/index.html:29]:
```suggestion
            <li class="nav__item">
              <a href="#apple" class="nav__link is-active">Apple</a>
            </li>
            
            <li class="nav__item">
              <a href="#samsung" class="nav__link">Samsung</a>
            </li>
            
            <li class="nav__item">
```

Add blank line between multiple sibling blocks of html

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,52 @@
...
+              <a href="#smartphones" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="hover">Laptops & Computers
+              </a>
```
Comment (nazarmatsevych) [inline src/index.html:37]:
```suggestion
              <a
                href="#laptops&computers"
                class="nav__link"
                data-qa="hover"
              >
                Laptops & Computers
              </a>
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,81 @@
...
+
+h1 {
+  position: absolute;
+  clip: rect(1px 1px 1px 1px);
+  padding: 0;
+  border: 0;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+}
```
Comment (nazarmatsevych) [inline src/style.css:24]: what for you need this styles?

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Almost done, pls check comments

File: src/style.css (hunk 3 above)
Comment (Igor-Pysak) [reply src/style.css:24]: > what for you need this styles?

For hide h1, or is it not necessary?

**Review by nazarmatsevych — APPROVED** [review]

**Pull request closed without merging**